woocommerce/woocommerce-android
# Issue/6428 replace manuals icons

Closes: #6428 


### Description


This PR adds the icons provided by the design team (context: p91TBi-7SX-p2#comment-9922 and p1649170847982409-slack-CGPNUU63E ) and replaces the generic manual icon with the correct icons on both the CardReaderManualsViewModel and CardReaderManualsViewModelTest files. 

### Testing instructions


- run the app in debug (use a store that has IPP enabled)
- Go to the Menu -> Settings -> In-Person Payments Tab
- Click on the Card Reader Manuals tab 
- Make sure the correct Card Reader Icons are displayed

### Images/gif


| Before | After |
| ---- | ---- |
| <img src="https://user-images.githubusercontent.com/30724184/167417844-b8c4deaa-10a1-4b83-91cb-34c231e75d17.png" width="350"/> | <img src="https://user-images.githubusercontent.com/30724184/167417906-ea39853c-5bd2-4ce9-9ba9-a14fbb194391.png" width="350"/> |
| <img src="https://user-images.githubusercontent.com/30724184/167417950-e4b70483-e49d-4451-a07e-346e95f2083b.png" width="350"/> | <img src="https://user-images.githubusercontent.com/30724184/167417992-043f91be-7f7a-44b6-ae20-fbf21a99889d.png" width="350"/> |
